demos/BMW_demo/bmw_share.js

```diff
@@ -0,0 +1,164 @@
+
+function bmw_compute_share(jiff,input,n, threshold, Zp) {
+  var ls={};// potential shares of length n
+  var i;
+  for (i=1;i<=n-1;i++) {
+    var b=Math.floor(Math.random()*2); // random from 0,1
+    ls[i]=b;
+
+  }
+  var sum=ls[1];
+  for (i=2;i<=n-1;i++) {
+    sum=sum^ls[i];
+  }
+  sum=sum^input;
+  ls[n]=sum;
+  //console.log("my compute for "+jiff.id);
+  //console.log(ls);
+
+  return ls;
+
+}
+
+
+// share the secret to corresponding party i with corresponding share[i]
```

Comment:
Same idea re: docs. Just to use the standard jiff Docstring format. this will allow JSDoc to properly generate the documentation pages (e.g. https://multiparty.org/jiff/docs/jsdoc/module-jiff-client-JIFFClient.html#share)